.github/workflows/python-package.yml

```diff
@@ -185,6 +185,9 @@ jobs:
     needs: isolated_tests
     if: startsWith(github.ref, 'refs/tags/')
     runs-on: ubuntu-latest
+    environment:
+      name: deploy
+      url: https://pypi.org/project/delocate/
```

Comment:
FYI you can also append `${{ github.ref_name}}` here for nicer URLs.

Reply:
That could work. I remember the first examples using the PyPI URL, but it's hard to tell what's most relevant.

Reply:
Well, I wrote those examples and never added this bit. But I use it in real-life projects (though, the workflows are more complicated and I wanted to keep the tutorial simple).

Reply:
Also, the reason there's `startsWith()` is that `ref_name` didn't exist at the time of writing and GH added it later.

Reply:
I'm going to have to update *a lot* of projects.